format-text/pom.xml

```diff
@@ -61,6 +61,11 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
+       <groupId>io.cdap.cdap</groupId>
+       <artifactId>hydrator-test</artifactId>
+       <scope>compile</scope>
```

Comment:
plugins should never have hydrator-test as compile scope, it should always be test scope.

Reply:
Done